components/IndicatorStat/IndicatorStatDataHandler.tsx

```diff
@@ -8,20 +8,44 @@ interface DataHandlerProps {
   dataLoading?: boolean;
   valueOptions?: ValueOptions;
   note?: ContentNote;
+  decimalCount?: number;
 }
 
-const IndicatorStatDataHandler: FunctionComponent<DataHandlerProps> = ({ data, dataLoading, ...props }) => {
+const IndicatorStatDataHandler: FunctionComponent<DataHandlerProps> = ({
+  data,
+  dataLoading,
+  decimalCount,
+  ...props
```

Comment:
This already takes in ValueOptions. Won't need decimalCount on its own.

Reply:
updated